### ISSUE TITLE
Suggestion | Add new category

### ISSUE THREAD

**Issue by SSergey911** [graphprotocol/mission-control-curator#237]
There are so many influential people and outstanding person who contributed to the building Web3 and bright Decentralized future in the Crypto World. We should show them some respect. Add the category **Person**. Also addition link related person to the projects is good option.

{
"name": "Person",
"slug": "person",
"description": "prominent and valuable people"
},

**Comment by yanivtal** [comment]
I agree **People** are going to be really important. I think of it as a separate subgraph / registry. Could be one of the next ones we focus on.

**Comment by SSergey911** [comment]
> I agree **People** are going to be really important. I think of it as a separate subgraph / registry. Could be one of the next ones we focus on.

This is a huge section in the entire crypto industry. Separate subgraph / registry looks very good!
